### ISSUE TITLE
Perioral erythema

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6243]
**Preferred term label:**
Perioral erythema

also 
Perianal erythema

**Synonyms**

**Definition (free text, please give PubMed ID)**
Erythema (Redness of the skin caused by hyperemia of the capillaries in the lower layers of the skin) localized to the region surrounding the mouth

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0010783

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:614328 

2/2 patients (both terms), https://pubmed.ncbi.nlm.nih.gov/22010916/  PMID:22010916

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
